src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java

```diff
@@ -57,9 +57,6 @@
  * This class is also used by {@link RecordedObject#toString()}
  */
 public final class PrettyWriter extends EventPrintWriter {
-    private static final Duration MILLSECOND = Duration.ofMillis(1);
```

Comment:
The original PR changes the copyright date at the top of this file. I did not include this change.

Reply:
Hmm, I'm not sure what the coding convention in OpenJDK is for updating copyright dates. Can you please check on that? We may want to preserve the update from the original PR (or possibly even use 2022).

Reply:
I'd recommend pulling in the new date (that's the unspoken convention)

Reply:
I copied the original backport with the copyright. Now this is an entirely clean backport.